### PR TITLE
Check-in view created with out functionalities

### PR DESCRIPTION
**Pull request by M-Cordova-M**
# Description

We added a check-in view, and tried some styles for the page we also added a clock with date and a form with text fields, it doesnt have functionalities, just styles.

## Screenshots

reloj 3

### CODE REVIEW

**Review by juperameza — APPROVED** [review]
Lgtm

**Review by MarcoRodriguez23 — APPROVED** [review]
You can create folders to save your code following the sass 7-1 guide https://sass-guidelin.es/#the-7-1-pattern

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -13,3 +13,107 @@
...
+  .checkin_header nav{
+    padding-top: 10px;
+    padding-bottom: 10px;
+    margin-right: 50px;
+    margin-left: 50px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center; 
+    width: 90%;
```
Comment (dagp-ucol) [inline app/assets/stylesheets/application.css:31]: Alphabetize your CSS

```suggestion
    align-items: center;
    display: flex;
    flex-direction: row;
    justify-content: space-between;
    margin-left: 50px;
    margin-right: 50px;
    padding-bottom: 10px;
    padding-top: 10px;
    width: 90%;
```

File: app/assets/stylesheets/application.css [hunk 2]
```diff
@@ -13,3 +13,107 @@
...
+  form button {
+    padding-top: 5px;
+    padding-bottom: 5px;
+    background-color: #007bff;
+    color: white;
+    border-style: hidden;
+    display: block;
+    width: 120px;
+    border-radius: 3px;
+
```
Comment (dagp-ucol) [inline app/assets/stylesheets/application.css:114]: Remove blank line
```suggestion
```

File: app/assets/stylesheets/application.css [hunk 3]
```diff
@@ -13,3 +13,107 @@
...
+  form button {
+    padding-top: 5px;
+    padding-bottom: 5px;
+    background-color: #007bff;
+    color: white;
+    border-style: hidden;
+    display: block;
+    width: 120px;
+    border-radius: 3px;
```
Comment (dagp-ucol) [inline app/assets/stylesheets/application.css:113]: Alphabetize your CSS

https://www.linkedin.com/pulse/alphabetize-your-css-get-out-dennis-plucinik/

File: app/assets/stylesheets/application.css [hunk 4]
```diff
@@ -13,3 +13,107 @@
...
+    margin-bottom: 35px;
+  }
+
+  #digital-clock {
+    width: 100%;
+    text-align: center;
+    font-size: 120px;
+    color: black;
+    margin-bottom: 0%;
+    height: 140px;
```
Comment (dagp-ucol) [inline app/assets/stylesheets/application.css:81]: Alphabetize your CSS

https://www.linkedin.com/pulse/alphabetize-your-css-get-out-dennis-plucinik/

File: app/assets/stylesheets/application.css [hunk 5]
```diff
@@ -13,3 +13,107 @@
...
+    height: 100%;
+  }
+  
+  #current_date {
+    font-weight: bold;
+    font-size: 30px;
+    text-align: center;
+    display: flex;   
+    justify-content: center; 
+    margin-bottom: 35px;
```
Comment (dagp-ucol) [inline app/assets/stylesheets/application.css:72]: Alphabetize your CSS

https://www.linkedin.com/pulse/alphabetize-your-css-get-out-dennis-plucinik/

File: app/assets/stylesheets/application.css [hunk 6]
```diff
@@ -13,3 +13,107 @@
...
+
+  .checkin_header nav h1{
+    color: white;
+  }
+
+  #title h1{
+    margin-top: 10px;
+    text-align: center;
+    font-size: 80px;
+    width: 100%;
```
Comment (dagp-ucol) [inline app/assets/stylesheets/application.css:59]: Alphabetize your CSS

https://www.linkedin.com/pulse/alphabetize-your-css-get-out-dennis-plucinik/

File: app/assets/stylesheets/application.css [hunk 7]
```diff
@@ -13,3 +13,107 @@
...
+
+  .checkin_header nav button{
+    padding-top: 5px;
+    padding-bottom: 5px;
+    background-color: #244c94;
+    color: white;
+    border-style: hidden;
+    display: block;
+    width: 120px;
+    border-radius: 3px;
```
Comment (dagp-ucol) [inline app/assets/stylesheets/application.css:42]: Alphabetize your CSS

https://www.linkedin.com/pulse/alphabetize-your-css-get-out-dennis-plucinik/

**Review by dagp-ucol — CHANGES_REQUESTED** [review]

**Review by oscaregr — APPROVED** [review]
made the nesesary chanfes

**Review by dagp-ucol — APPROVED** [review]

**Pull request merged**